workers/data_refinery_workers/processors/smasher.py

```diff
@@ -133,8 +135,8 @@ def _smash(job_context: Dict) -> Dict:
 
         # Metadata to TSV
         with open(smash_path + 'metadata.tsv', 'w') as output_file:
-            sample_ids = [k for k in metadata['samples'].keys()]
-            keys = [k for k in metadata['samples'][sample_ids[0]].keys()] + ['sample_id']
+            sample_ids = list(metadata['samples'].keys())
+            keys = list(metadata['samples'][sample_ids[0]].keys()) + ['sample_id']
```

Comment:
The list comprehension wasn't the confusing part of this syntax. It was `metadata['samples'][sample_ids[0]].keys()` which is so many layers of lookups at once it's very hard to tell what is happening, especially without having the structure of `metadata` fresh in my head. If you can find a way to clarify this cool, if not it's not that big of a deal.